packages/duktape/build.sh

```diff
@@ -0,0 +1,36 @@
+TERMUX_PKG_HOMEPAGE=https://www.duktape.org/
+TERMUX_PKG_DESCRIPTION="The Duktape JavaScript interpreter"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_VERSION=2.3.0
+TERMUX_PKG_SKIP_SRC_EXTRACT=1
+TERMUX_PKG_BUILD_IN_SRC=yes
+
+termux_step_extract_package() {
+	git clone --depth=1 https://github.com/svaarala/duktape.git -b v${TERMUX_PKG_VERSION} ${TERMUX_PKG_SRCDIR}
+}
+
+termux_step_post_extract_package() {
+	sudo apt-get -yq update
+	sudo apt-get install -yq python2.7 python-pip
+	pip install pyyaml
+}
+
+termux_step_make() {
+	make libduktape.so.1.0.0 duk CC=${CC} GXX=${CXX}
+}
+
+termux_step_make_install() {
+	cp libduktape.so.1.0.0 ${TERMUX_PREFIX}/lib/libduktape.so
+	cp duk ${TERMUX_PREFIX}/bin
+}
```

Comment:
You are installing only library and program. Headers are not needed here ?
Something like
```
include/duk_config.h
include/duktape.h
```

Reply:
Done!